src/settings/ActionProfiles/AssociatedJobProfiles.js

```diff
@@ -0,0 +1,234 @@
+        nsParams={associatedJobProfilesNsParams}
+        initialSearchState={initialQuery}
+      >
+        {({
+          searchValue,
+          getSearchHandlers,
+          onSubmitSearch,
+          onSort,
+          resetAll,
+        }) => {
+          const onSubmit = event => {
+            onSubmitSearch(event);
+            deselectAll();
+          };
+          const searchButtonDisabled = !searchValue.query;
+
+          return (
+            <Fragment>
+              <form onSubmit={onSubmit}>
+                <div className={searchAndSortCss.searchWrap}>
+                  <div className={searchAndSortCss.searchFiledWrap}>
+                    <SearchField
+                      id="input-associated-job-profiles-search"
+                      clearSearchId="input-associated-job-profiles-clear-search-button"
+                      loading={!associatedJobProfilesResource.hasLoaded}
+                      aria-label="associated job profiles search"
+                      marginBottom0
+                      name="query"
+                      value={searchValue.query}
+                      onChange={getSearchHandlers().query}
+                      onClear={resetAll}
+                    />
+                  </div>
+                  <div className={searchAndSortCss.searchButtonWrap}>
+                    <Button
+                      data-test-search-and-sort-submit
+                      disabled={searchButtonDisabled}
+                      type="submit"
+                      buttonStyle="primary"
+                      fullWidth
+                      marginBottom0
+                    >
+                      <FormattedMessage id="stripes-smart-components.search" />
+                    </Button>
+                  </div>
+                </div>
+              </form>
+              <IntlConsumer>
+                {intl => (
+                  <MultiColumnList
+                    id="associated-job-profiles-list"
+                    visibleColumns={associatedJobProfilesVisibleColumns}
+                    contentData={associatedJobProfiles}
+                    columnMapping={{
+                      selected: (
+                        <div // eslint-disable-line jsx-a11y/click-events-have-key-events
+                          role="button"
+                          tabIndex="0"
+                          className={sharedCss.selectableCellButton}
+                          data-test-select-all-associated-job-profiles-checkbox
+                          onClick={e => e.stopPropagation()}
+                        >
+                          <Checkbox
+                            name="selected-all"
+                            checked={isAllSelected}
+                            onChange={handleSelectAllCheckbox}
+                          />
+                        </div>
+                      ),
+                      name: intl.formatMessage({ id: 'ui-data-import.name' }),
```

Comment:
Just for curiosity, I think now columns strings can be passed as FormattedMessage components. Though it still requires a check.

Reply:
That is true. Changed it everywhere.

Reply:
Great.